### PR TITLE
Folded likes

### LINKED ISSUE #24: Show 3 latest likes and fold others to "X other people" link
Show 3 latest likes and fold others to "X other people" link. If a user clicks on that link the application will expand it and show other users who likes that post.

### PR DESCRIPTION
**Pull request by berkus**
Fixes #24 

With minor deficiency from FRF:
- After Un-liking the showAllLikes is not reset to false as it should.

Potential improvement:
- Current user's like can be put always first, see commented code.

### CODE REVIEW

File: public/js/app/models/Post.js [hunk 1]
```diff
@@ -19,21 +20,13 @@ define(["config",
...
 
-      // TODO: refactor to mapProperty
-      likes.forEach(function(like) {
-        if (like.id === App.properties.userId) {
-          liked = true;
-          return true;
-        }
-      })
-      return liked;
+      return likes.isAny('id', /*==*/ App.properties.userId)
```
Comment (epicmonkey) [inline public/js/app/models/Post.js]: I found an easter egg!

File: public/js/app/models/Post.js [hunk 2]
```diff
@@ -46,6 +39,38 @@ define(["config",
...
+      var likes = this.get('likes')
+      if (!likes) return;
+
+      var partLikes = Ember.A([])
+
+      // Put current user's like always first
+      /*if (this.get('currentUserLiked')) {
+        partLikes.pushObjects(likes.filterBy('id', App.properties.userId))
+        partLikes.pushObjects(likes.rejectBy('id', App.properties.userId).objectsAt([0, 1]))
+      } else*/ {
```
Comment (epicmonkey) [inline public/js/app/models/Post.js]: Let's drop commented out code.

File: public/js/app/models/Post.js (hunk 1 above)
Comment (berkus) [inline public/js/app/models/Post.js]: Wha?

File: public/js/app/models/Post.js (hunk 2 above)
Comment (berkus) [inline public/js/app/models/Post.js]: ✅

**Pull request merged**